samples/ChangeFeedManagement/Program.py

```diff
@@ -98,11 +98,11 @@ def run_sample():
                 if e.status_code == 409:
                     pass
                 else:
-                    raise errors.HTTPFailure(e.status_code)
+                    raise
 
             # setup collection for this sample
 
-            collection_definition = {   'id': CONTAINER_ID, 
+            collection_definition = {   'id': COLLECTION_ID,
```

Comment:
This would have `NameError`-ed, since `CONTAINER_ID` was never instantiated anywhere.